src/components/contextual/pages/pool/staking/StakingIncentivesCard.vue

```diff
@@ -167,7 +173,16 @@ function handlePreviewClose() {
                     <BalTooltip :text="$t('staking.unstakedLpTokensTooltip')" />
                   </BalStack>
                 </BalStack>
-                <BalStack horizontal spacing="sm" class="mt-2">
+                <BalBtn
+                  v-if="hasNonPrefGaugeBalance && !isAffected"
```

Comment:
Why do we use isAffected here? Can you explain what situation it prevents?

Reply:
not sure here, it is copy of old code, I decided not to change it